packages/client/src/pages/settings/sounds.vue

```diff
@@ -27,6 +27,28 @@ import * as os from '@/os';
 import { ColdDeviceStorage } from '@/store';
 import { playFile } from '@/scripts/sound';
 import * as symbols from '@/symbols';
+import { i18n } from '@/i18n';
+
+const sounds = ref<any>({});
+
+const masterVolume = computed({
+	get: () => {
+		return ColdDeviceStorage.get('sound_masterVolume');
+	},
+	set: (value) => {
+		ColdDeviceStorage.set('sound_masterVolume', value);
+	}
+});
+
+const volumeIcon = computed(() => masterVolume.value === 0 ? 'fas fa-volume-mute' : 'fas fa-volume-up');
+
+sounds.value.note = ColdDeviceStorage.get('sound_note');
+sounds.value.noteMy = ColdDeviceStorage.get('sound_noteMy');
+sounds.value.notification = ColdDeviceStorage.get('sound_notification');
+sounds.value.chat = ColdDeviceStorage.get('sound_chat');
+sounds.value.chatBg = ColdDeviceStorage.get('sound_chatBg');
+sounds.value.antenna = ColdDeviceStorage.get('sound_antenna');
+sounds.value.channel = ColdDeviceStorage.get('sound_channel');
```

Comment:
I think the initialization of `sounds` could be merged here, something like this.

```suggestion
const sounds = ref({
    note: ColdDeviceStorage.get('sound_note'),
    noteMy: ColdDeviceStorage.get('sound_noteMy'),
    notification: ColdDeviceStorage.get('sound_notification'),
    chat: ColdDeviceStorage.get('sound_chat'),
    chatBg: ColdDeviceStorage.get('sound_chatBg'),
    antenna: ColdDeviceStorage.get('sound_antenna'),
    channel: ColdDeviceStorage.get('sound_channel'),
});
```

(and remove the earlier assignment to `sounds`)